plugins/BEdita/Core/src/Model/Behavior/SearchableBehavior.php

```diff
@@ -159,8 +159,25 @@ function ($word) use ($minLength) {
             ]);
         }
 
+        $table = $this->getTable();
+        $tables = array_reverse(
+            array_merge(
+                [$table],
+                ($table instanceof InheritanceTable) ? $table->inheritedTables() : []
+            )
+        );
+        $aliasField = function ($field) use ($tables) {
+            /* @var \Cake\ORM\Table $table */
+            foreach ($tables as $table) {
+                if ($table->hasField($field, false)) {
+                    return $table->aliasField($field);
+                }
+            }
+
+            return $field;
```

Comment:
This was an edge case, so hopefully this line was never hit. However, I have slightly refactored the closure, and now coverage should be 100%. 🎉